packages/select/src/index.mts

```diff
@@ -78,12 +78,11 @@ export default createPrompt(
     }, [items]);
 
     const defaultItemIndex = useMemo(() => {
-      if (!_default) return -1;
+      if (!('default' in config)) return -1;
```

Comment:
agree that it's nicer, but perhaps we should just rename all `default` to `defaultValue` as it's a bit annoying to work with the reserved word imo.